backend/corpora/common/entities/project.py

```diff
@@ -0,0 +1,146 @@
+import typing
```

Comment:
Nit for this file, can we avoid using single character names; usually is good practice to fully write out the name for readability (k = key, v = value, etc.).